tools/tidb-controller.md

```diff
@@ -0,0 +1,110 @@
+
+## Usage introduction
+
+The usage of `tidb-ctl` consists of command (including subcommand), option, and flag.
+
+- command: characters without `-` or `--`
+- option: characters with `-` or `--`
+- flag: characters exactly following the command or option, passing value to the command or option
+
+Usage example: `tidb-ctl schema in mysql -n db`
+
+- `schema`: the command
+- `in`: the subcommand of schema
+- `mysql`: the flag of `in`
+- `-n`: the option
+- `db`: the flag of `-n`
+
+### Get help
+
+Use `tidb-ctl -h/--help` to get the help information. `tidb-ctl` consists of multiple layers of commands. You can use `-h/--help` to get the help information of `tidb-ctl` and all other subcommands.
+
+### Connect
+
+```
+tidb-ctl -H/--host {TiDB service address} -P/--port {TiDB service port}
+```
+
+If you do not add an address or a port, the default value is used. The default address is `127.0.0.1` (service address must be the IP address); the default port is `10080`. Connection options are top-level options and apply to all of the following commands.
+
+Currently, TiDB Controller can obtain four categories of information using the following four commands:
+
+- `tidb-ctl mvcc`: MVCC information
+- `tidb-ctl region`: Region information
+- `tidb-ctl schema`: Schema information
+- `tidb-ctl table`: Table information
+
+### Examples
+
+The following example shows how to obtain the schema information:
+
+Use `tidb-ctl schema -h` to get the help information of the subcommands. `schema` has two subcommands: `in` and `tid`.
+
+- `in` is used to obtain the table schema of all tables in the database through the database name.
+- `tid` is used to obtain the table schema through the unique `table_id` in the whole database.
+
+#### The `in` command
+
+You can also use `tidb-ctl schema in -h/--help` to get the help information of the `in` subcommand.
+
+##### Basic usage
+
+```
+tidb-ctl schema in {database name}
+```
+
+For example, `tidb-ctl schema in mysql` returns the following result:
+
+```text
+[
+    {
+        "id": 13,
+        "name": {
+            "O": "columns_priv",
+            "L": "columns_priv"
+        },
+              ...
+        "update_timestamp": 399494726837600268,
+        "ShardRowIDBits": 0,
+        "Partition": null
+    }
+]
+```
+
+The result is long and displayed in JSON. The above result is a truncated one.
+
+- If you want to specify the table name, use `tidb-ctl schema in {database} -n {table name}` to filter.
+
+    For example, `tidb-ctl schema in mysql -n db` returns the table schema of the `db` table in the `mysql` database:
+
+    ```text
+    {
+        "id": 9,
+        "name": {
+            "O": "db",
+            "L": "db"
+        },
+        ...
+        "Partition": null
+    }
+    ```
+
+    The above result is a truncated one, too.
+
+- If you want to specify the server name, use the `-H -P` option.
```

Comment:
Use `server address` or sth is better. This tool does not support to use the server name directly. Only IP + port is available.